docs/dev/app/listings.md

```diff
@@ -1,9 +1,416 @@
 # Listing Pages
 
-Basic structure; feel free to deviate.
+All three listing pages contain a filter bar at the top, a scrollable list of Listing Cards on the left, and the Listing Details of the currently selected listing card on the right. Differences in display between listing pages are listed below. Then, the usage and functionality of components used to bulid the listing pages are explained. 
 
-## Cases
+## Listing Page Comparison 
 
-## Limited Case Assignments
+<style>
+table th:first-of-type {
+    width: 15%;
+}
+table th:nth-of-type(2) {
+    width: 30%;
+}
+table th:nth-of-type(3) {
+    width: 25%;
+}
+table th:nth-of-type(4) {
+    width: 25%;
+}
+</style>
 
-## Language Support
+| Listing Page | Case | Limited Case Assignments | Language Support |
+|:------------:|------|--------------------------|------------------|
+| Filters | <ol><li>Remote/In Person</li><li>Languages</li><li>Adjudicating Agency</li><li>Country of Origin</li></ol> | <ol><li>Country Field</li><li>Language(s)</li></ol> | <ol><li>Listing Type</li><li>Language(s)</li></ol> | <ol><li>Listing Type</li><li>Language(s)</li></ol> | <ol><li>Listing Type</li><li>Language(s)</li></ol> |
+| Listing Highlights | <ol><li>Relief sought</li><li>Time Commitment</li><li>Remote/In Person</li><li>Adjudicating Agency</li><li>Client Languages</li><li>Client Country of Origin</li><li>__*__ Client Location</li></ol> | <ol><li>Country Field</li><li>Language(s)</li><li>Expected Deliverable</li></ol> | **DOC** <ol><li>Language(s)</li><li>Number of Pages</li></ol> **CASE_INT** <ol><li>Language(s)</li><li>Time Commitment</li><li> Remote/In Person </li></ol> **INT** <ol><li>Language(s) </li><li>Remote/In Person</li></ol> |
+
+::: note (*) Note: 
+If the client is in custody (i.e., `is_detained`, pulled from LegalServer, is `true`), "Client Location" appears as "Custody Location." Otherwise, it remains as Client Location.
+::: 
+
+## **Components Used**
+
```

Comment:
can we add a screenshot here? Lemme send it over!

Reply:
added!

Reply:
removed ListingCard image!